go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/docker/compose/v2
 
 go 1.18
 
+replace github.com/cucumber/godog => github.com/laurazard/godog v0.0.0-20220922095256-4c4b17abdae7
```

Comment:
What's behind the need for this replace, out of curiosity?

Reply:
`cucumber/godog` currently allows for passing in a `testing.T` instance to run scenarios as subtests with, but it does not make that `testing.T` instance available to the scenario initializer, which was needed for passing to our `CLI`. I'll make a PR to see if I can contribute that upstream soon :)